service/src/main/java/uk/gov/hmcts/reform/fpl/validators/HasApplicantContactNameValidator.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.hmcts.reform.fpl.validators;
+
+import org.apache.commons.lang.StringUtils;
+import uk.gov.hmcts.reform.fpl.model.ApplicantParty;
+import uk.gov.hmcts.reform.fpl.validators.interfaces.HasApplicantContactName;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+
+public class HasApplicantContactNameValidator implements ConstraintValidator<HasApplicantContactName, ApplicantParty> {
+    @Override
```

Comment:
No need to override it if empty. Interface comes with default implementation. Please remove.

Reply:
Let's just remove the initialise function as we do not use

Reply:
This is exactly what I suggested James.